### PR TITLE
Titan

### PR DESCRIPTION
**Pull request by annaweber209**
makes training_affinity.py compatible with pytoda 0.2.4 (smiles language handling)
clearer nomenclature and differentiation of input channels in bimodal_mca
supports use of different embeddings in bimodal_mca
supports use of temperature hyperparameter in bimodal mca

### CODE REVIEW

File: examples/affinity/train_affinity.py [hunk 1]
```diff
@@ -88,19 +88,58 @@ def main(
     device = get_device()
 
     # Load languages
-    smiles_language = SMILESLanguage.load(smiles_language_filepath)
-    protein_language = ProteinLanguage.load(protein_language_filepath)
+    smiles_language_filepath = os.path.join(
+        '/',
```
Comment (jannisborn) [inline examples/affinity/train_affinity.py]:
```suggestion
        os.sep,
```

File: examples/affinity/train_affinity.py [hunk 2]
```diff
@@ -88,19 +88,58 @@ def main(
...
+        *metadata.__file__.split('/')[:-1], 'smiles_language'
```
Comment (jannisborn) [inline examples/affinity/train_affinity.py]:
```suggestion
        *metadata.__file__.split(os.sep)[:-1], 'smiles_language'
```

**Review by jannisborn — COMMENTED** [review]
Very good. I think we can almost merge!
Just two small comments:

- we should increase the package version in `__init__.py` to 0.0.3. The changes in the bimodall MCA are refactors but the compatibility cant be ensured.
- now `training_affinity.py` will always use the pytoda build-in smiles language. I think it should still be optional to pass a SMILES language. With the new notation and the `.from_pretrained` function, the difference is that the path needs to point to a folder with the designated json scripts, just like in the `pytoda.smiles.metadata`. The argument could be optional, s.t. the pytoda language is loaded per default.

**Comment by annaweber209** [comment]
Alright, version is changed and option to specify a SMILES language file path is implemented! :)

**Review by jannisborn — APPROVED** [review]

**Comment by jannisborn** [comment]
Great stuff @annaweber209. Let me just remove the Travis checks (we should migrate CI to github actions soon...)

**Pull request merged**